on_device_training/mobilebert/mobilebert_offline.ipynb

```diff
@@ -0,0 +1,485 @@
t', 'mobilebert.encoder.layer.14.bottleneck.attention.LayerNorm.weight', 'mobilebert.encoder.layer.7.intermediate.dense.weight', 'mobilebert.encoder.layer.18.ffn.0.output.dense.weight', 'mobilebert.encoder.layer.19.attention.self.key.bias', 'mobilebert.encoder.layer.10.attention.self.value.weight', 'mobilebert.encoder.layer.3.bottleneck.attention.LayerNorm.weight', 'mobilebert.encoder.layer.18.ffn.2.output.dense.weight', 'mobilebert.encoder.layer.23.ffn.2.intermediate.dense.bias', 'mobilebert.encoder.layer.7.attention.self.value.weight', 'mobilebert.encoder.layer.3.ffn.1.intermediate.dense.weight', 'mobilebert.encoder.layer.3.ffn.1.output.dense.bias', 'mobilebert.encoder.layer.6.ffn.0.output.LayerNorm.weight', 'mobilebert.encoder.layer.12.bottleneck.input.dense.bias', 'mobilebert.encoder.layer.4.ffn.0.output.LayerNorm.weight', 'mobilebert.encoder.layer.15.ffn.1.output.dense.bias', 'mobilebert.encoder.layer.2.output.bottleneck.dense.weight', 'mobilebert.encoder.layer.14.attention.self.key.bias', 'mobilebert.encoder.layer.23.ffn.1.intermediate.dense.bias', 'mobilebert.encoder.layer.6.output.bottleneck.LayerNorm.weight', 'mobilebert.encoder.layer.11.output.bottleneck.LayerNorm.weight', 'mobilebert.encoder.layer.19.ffn.0.output.LayerNorm.weight', 'mobilebert.encoder.layer.11.attention.self.key.bias', 'mobilebert.encoder.layer.16.output.bottleneck.dense.bias', 'mobilebert.encoder.layer.7.ffn.2.intermediate.dense.bias', 'mobilebert.encoder.layer.8.ffn.2.output.LayerNorm.weight', 'mobilebert.encoder.layer.8.ffn.2.intermediate.dense.bias', 'mobilebert.encoder.layer.17.bottleneck.attention.dense.bias', 'mobilebert.encoder.layer.12.ffn.0.output.dense.bias', 'mobilebert.encoder.layer.19.attention.self.key.weight', 'mobilebert.encoder.layer.6.bottleneck.input.dense.weight', 'mobilebert.encoder.layer.20.ffn.0.intermediate.dense.bias', 'mobilebert.encoder.layer.7.ffn.1.output.LayerNorm.weight', 'mobilebert.encoder.layer.13.ffn.0.output.LayerNorm.weight', 'mobilebert.encoder.layer.20.bottleneck.input.LayerNorm.bias', 'mobilebert.encoder.layer.7.bottleneck.attention.LayerNorm.weight', 'mobilebert.encoder.layer.12.ffn.2.output.dense.weight', 'mobilebert.encoder.layer.11.output.LayerNorm.bias', 'mobilebert.encoder.layer.23.attention.output.dense.weight', 'mobilebert.encoder.layer.10.ffn.1.output.LayerNorm.bias', 'mobilebert.encoder.layer.3.ffn.0.output.LayerNorm.weight', 'mobilebert.encoder.layer.11.ffn.2.output.LayerNorm.bias', 'mobilebert.encoder.layer.12.attention.output.dense.bias', 'mobilebert.encoder.layer.19.bottleneck.input.LayerNorm.bias', 'mobilebert.encoder.layer.15.ffn.1.output.LayerNorm.bias', 'mobilebert.encoder.layer.20.output.bottleneck.dense.bias', 'mobilebert.encoder.layer.15.intermediate.dense.weight', 'mobilebert.encoder.layer.20.intermediate.dense.bias', 'mobilebert.encoder.layer.13.bottleneck.input.LayerNorm.weight', 'mobilebert.encoder.layer.16.ffn.1.output.LayerNorm.weight', 'mobilebert.encoder.layer.17.ffn.0.intermediate.dense.weight']\n",
```

Comment:
Let's remove the stdout from the python notebook.

Reply:
Oops, just removed!